geniza/corpus/models.py

```diff
@@ -524,7 +527,7 @@ def index_data(self):
         index_data.update(
             {
                 "pgpid_i": self.id,
-                "type_s": self.doctype.name if self.doctype else "Unknown",
+                "type_s": str(self.doctype) if self.doctype else "Unknown type",
```

Comment:
We _might_ need to differentiate between the actual document type and the display label for the type — we're going to have a facet/filter on document type and I'm not sure which one makes most sense in that context. Seems reasonable to assume for now that they should be consistent, so let's leave this for now.